src/constants/commonConstants.ts

```diff
@@ -28,21 +19,15 @@ export const DEFAULT_USER_STATE = {
   password: '',
 };
 
-export const MOCK_USER_STATE = {
-  first_name: 'Ivan',
-  second_name: 'Petrov',
-  email: 'email1@email.com',
-  login: 'user1',
-  phone: '+79919927812',
-  password: '123456',
-};
+export type userType = typeof DEFAULT_USER_STATE;
+
+export type userKeys = keyof userType;
```

Comment:
❗️
Константы должны именоваться одинаково везде
```suggestion
export type USER_TYPE = typeof DEFAULT_USER_STATE;

export type USER_KEYS = keyof userType;
```

Reply:
я думал что к типам это не применяется =)

Reply:
fixed

Reply:
Если они лежат в константах, то именоваться должны как константы. Дискуссионный вопрос — должны ли они лежать константах. Но это уже совсем другая история)